bridge-token-factory/tests/token_transfer.rs

```diff
@@ -1,51 +1,73 @@
+use bridge_token_factory::{validate_eth_address, EthLockedEvent, Proof};
 use near_sdk::borsh::{self, BorshSerialize};
+use near_sdk::{AccountId, Balance, ONE_NEAR, ONE_YOCTO};
 use serde_json::json;
+use tokio::runtime::Runtime;
+use workspaces::prelude::*;
+use workspaces::result::CallExecutionDetails;
+use workspaces::{network::Sandbox, Account, Contract, Worker};
 
-use bridge_token_factory::BridgeTokenFactoryContract;
-use bridge_token_factory::{validate_eth_address, EthLockedEvent, Proof};
-use mock_prover::MockProverContract;
-use near_sdk::json_types::ValidAccountId;
-use near_sdk_sim::runtime::GenesisConfig;
-use near_sdk_sim::{
-    call, deploy, init_simulator, units::to_yocto, view, ContractAccount, ExecutionResult,
-    UserAccount,
-};
-
-const PROVER: &str = "prover";
 const FACTORY: &str = "bridge";
 const LOCKER_ADDRESS: &str = "11111474e89094c44da98b954eedeac495271d0f";
 const DAI_ADDRESS: &str = "6b175474e89094c44da98b954eedeac495271d0f";
 const SENDER_ADDRESS: &str = "00005474e89094c44da98b954eedeac495271d0f";
-const ALICE: &str = "alice";
+const ALICE: &str = "alice.test.near";
 
-near_sdk_sim::lazy_static_include::lazy_static_include_bytes! {
-    TEST_TOKEN_WASM_BYTES => "../res/test_token.wasm",
-    TOKEN_WASM_BYTES => "../res/bridge_token.wasm",
-    FACTORY_WASM_BYTES => "../res/bridge_token_factory.wasm",
-    MOCK_PROVER_WASM_BYTES => "../res/mock_prover.wasm",
-}
+const FACTORY_WASM_PATH: &str = "../res/bridge_token_factory.wasm";
+const MOCK_PROVER_WASM_PATH: &str = "../res/mock_prover.wasm";
 
-fn setup_token_factory() -> (UserAccount, ContractAccount<BridgeTokenFactoryContract>) {
-    let mut config = GenesisConfig::default();
-    config.runtime_config.storage_amount_per_byte = 10u128.pow(19);
-    let root = init_simulator(Some(config));
-    let _prover = deploy!(
-        contract: MockProverContract,
-        contract_id: PROVER.to_string(),
-        bytes: &MOCK_PROVER_WASM_BYTES,
-        signer_account: root
-    );
-    let factory = deploy!(
-        contract: BridgeTokenFactoryContract,
-        contract_id: FACTORY.to_string(),
-        bytes: &FACTORY_WASM_BYTES,
-        signer_account: root,
-        init_method: new(
-            PROVER.to_string(), LOCKER_ADDRESS.to_string()
+const STORAGE_AMOUNT: u128 = 50_000_000_000_000_000_000_000_000;
```

Comment:
If we really need to use it for some reason, then it could be just `50 * ONE_NEAR`

Reply:
Not exactly. 1 NEAR is enough. But more accurate calculation is quite complicated and I don't think it is worth to do.

Reply:
https://github.com/aurora-is-near/rainbow-token-connector/pull/159/commits/18b96d78d388e8538c0544156d4be053a6919092